index.bs

```diff
@@ -2607,9 +2664,9 @@ const inputs = {
   'input2': inputBuffer2,
 };
 const outputs = {'output': outputBuffer};
-await context.compute(graph, inputs, outputs);
+const result = await context.compute(graph, inputs, outputs);
 
-console.log('Output value: ' + outputBuffer);
+console.log('Output value: ' + result.output);
```

Comment:
`result.ouputs.output`?

Reply:
Thanks for catching this bug! fixed in the latest commit.